src/components/Books.js

```diff
@@ -10,13 +10,13 @@ const Books = (props) => {
   const books = props.data;
   return (
     <div>
-      <h1>My Books</h1>
       <ul>
         {
         // eslint-disable-next-line react/prop-types
         books.map((book) => <Book key={book.item_id} bookArray={book} />)
         }
       </ul>
+      <hr />
```

Comment:
- [ ] Books section isn't horizontally aligned properly, kindly make sure to follow the design guide as much as possible.
![image](https://user-images.githubusercontent.com/33767951/140624509-d41c5682-d654-4235-8304-985ab0f9608f.png)